inst/templates/tiltWorkflows.Rmd

```diff
@@ -1,236 +1,207 @@
 ---
-title: "tilt workflow"
+title: "Creating outputs for all tilt indicators"
 author: "Kalash Singhal"
 ---
 
 ```{r, include = FALSE}
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>",
-  # FIXME when this file become reproducible
-  eval = FALSE
+  comment = "#>"
 )
 ```
 
-This file uses the output from the tiltIndicatorBefore package as the inputs to
-the tiltIndicator and tiltIndicatorAfter packages. This script uses the CSV
-files from the tiltIndicatorBefore package to create the final output of the
-four indicators.
+This workflow creates the output of all tilt indicators.
 
 ## Setup
 
 ```{r}
 library(dplyr, warn.conflicts = FALSE)
 library(readr, warn.conflicts = FALSE)
-library(tidyr, warn.conflicts = FALSE)
-# devtools::install_github("2DegreesInvesting/tiltIndicatorAfter")
-library(tiltIndicatorAfter)
-```
-
-## tiltIndicatorAfter data common for all indicators
-
-`ep_companies`, `ei_activities_overview`, `mapper_ep_ei`, `ei_input_data`, and
-`isic_4digit_name` are the five datasets which are used as the input for the
-tiltIndicatorAfter package. They all are the output from tiltIndicatorBefore
-package. These five datasets are used for adding additional data to all the four
-indicators (Emissions profile, Emissions profile upstream, Sector profile,
-Sector profile upstream).
-
-```{r}
-# ep_companies
-europages_campanies <- read_csv("/path/ep_companies.csv") |>
-  select("company_name", "country", "company_city", "postcode", "address", "main_activity", "companies_id")
-
-# ecoinvent_activities
-ei_activities_ov <- read_csv("/path/ei_activities_overview.csv")
-
-# matches_mapper
-mapper_ep_ei <- read_csv("/path/mapper_ep_ei.csv")
+library(fs)
+library(tiltWorkflows)
 
-# ecoinvent_inputs
-ei_input <- read_csv("/path/ei_input_data.csv") |>
-  select("input_activity_uuid_product_uuid", "exchange_name", "exchange_unit_name") |>
-  distinct()
+# Helper
+tilt_path <- function(...) {
+  path_home("Downloads", "tilt", ...)
+}
 
-# isic_4digit_name
-isic_4digit_name <- read_csv("/path/isic_4digit_name.csv")
+dir_create(tilt_path())
+options(width = 500)
 ```
 
-## Emissions profile
-
-### Load Data
+## Data
 
-The input data of Emissions_profile which comes from tiltIndicatorBefore package
-becomes the input for the tiltIndicator package.
+Here we use toy datasets that you should replace with your real data.
 
 ```{r}
-# emissions_profile companies
-ep_company <- read_csv("/path/emissions_profile_any_companies_ecoinvent.csv")
+europages_campanies <- tiltIndicatorAfter::ep_companies |>
+  # TODO: Ask why the toy data has more columns than we need
+  select(
+    "company_name",
+    "country",
+    "company_city",
+    "postcode",
+    "address",
+    "main_activity",
+    "companies_id"
+  )
```

Comment:
@kalashsinghal 
Can we remove the other columns (if any) from the toy data?

Reply:
@maurolepore There are no other columns from this dataset. Hence, you can just delete this column selection code here. :)

Reply:
But note this workflow file should work with both toy data and real data.  With this in mind, should I still remove the columns selection?

Reply:
@maurolepore Apologies! I just noticed that this column selection is needed for `ep_companies`. Hence, this code can't be removed.